tests/Doctrine/Tests/KeyValueStore/Storage/AmazonDynamoDbTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Doctrine\Tests\KeyValueStore\Storage;
+
+use Aws\DynamoDb\DynamoDbClient;
+use Aws\Result;
+use Doctrine\KeyValueStore\Storage\AmazonDynamoDbStorage;
+
+class AmazonDynamoDbTest extends \PHPUnit_Framework_TestCase
+{
+    private function getDynamoDbMock($methods = [])
+    {
+        $client = $this->getMockBuilder(DynamoDbClient::class)->disableOriginalConstructor();
```

Comment:
Use `'Aws\DynamoDb\DynamoDbClient'` instead of `DynamoDbClient::class` please, so we can keep the php 5.4 support also on dev. :)

The same for every other `::class` used below.